fix/fixer_pp_docker_tag_tags.go

```diff
@@ -0,0 +1,104 @@
+package fix
+
+import (
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+)
+
+// FixerDockerTagtoTags renames any Filename to Output
+type FixerDockerTagtoTags struct{}
+
+func (FixerDockerTagtoTags) Fix(input map[string]interface{}) (map[string]interface{}, error) {
+	if input["post-processors"] == nil {
+		return input, nil
+	}
+
+	// Our template type we'll use for this fixer only
+	type template struct {
+		PP `mapstructure:",squash"`
+	}
+
+	// Decode the input into our structure, if we can
+	var tpl template
+	if err := mapstructure.Decode(input, &tpl); err != nil {
+		return nil, err
+	}
+
+	// Go through each post-processor and get out all the complex configs
+	pps := tpl.ppList()
+
+	for _, pp := range pps {
+		ppTypeRaw, ok := pp["type"]
+		if !ok {
+			continue
+		}
+
+		if ppType, ok := ppTypeRaw.(string); !ok {
+			continue
+		} else if ppType != "docker-tag" {
+			continue
+		}
+
+		// Create a []string to hold tag and tags values
+		allTags := []string{}
+
+		tagRaw, ok := pp["tag"]
+		if ok {
+			// Gather all "tag" into the []string
+			switch t := tagRaw.(type) {
+			case []interface{}:
+				for _, tag := range t {
+					allTags = append(allTags, tag.(string))
+				}
+			case []string:
+				allTags = append(allTags, t...)
+			case string:
+				tList := strings.Split(t, ",")
```

Comment:
Should we strip white space here? Thinking in the case of `"tag": "one, two"`

Reply:
good point